### ISSUE TITLE
Graph functionality on Expenses 

### ISSUE THREAD

**Issue by friedrice123** [Cs-253-Team-4/Cs-253-Team-4-Evensplit#9]
Graph functionality on the Groups page under Expenses is not evenly distributed, the prices are not visible as they mix into each other because of no restrictions on control over the angles, distances between the nodes. In addition to that, the nodes vanish after you drag them to a side and they are not visible after that and there is no way to access them. Summarizing this, there is no bounded box for the graph.
![image](https://user-images.githubusercontent.com/103812705/230789442-a182bfa9-20e1-4a1d-aad7-27dd6eb520b9.png)
Default 
![image](https://user-images.githubusercontent.com/103812705/230789460-8b56ba66-919e-4972-bba3-c12b1ea6acd1.png)

**Comment by dhruvgarg004** [comment]
We used the d3 react library for the graphs. The CORRECTNESS of the graph is unquestionable. Coming to the untidy rendering, you might have guessed that we tried our best to space out the nodes, but the high level of randomness of the graphs stood in our way.

Also, we see that you have been unforgivingly reckless with the nodes of the graph in dragging them all the way out of the allocated division. Maybe we must also add instructions on "how to drag nodes desirably". As of now, we believe adding a bigger division for the graph would suffice.
